delocate/cmd/delocate_wheel.py

```diff
@@ -37,6 +45,8 @@ def main():
                action="store_true",
                help="Only analyze files with known dynamic library "
                "extensions"),
+        Option("-e", "--exclude",
```

Comment:
Or allow multiple `--exclude` options with `action="append"`?

Reply:
I've rewritten the code to use `action="append"` but I haven't tested it.